charts/open-webui/templates/workload-manager.yaml

```diff
@@ -15,6 +15,12 @@ metadata:
   {{- end }}
 spec:
   replicas: {{ .Values.replicaCount }}
+  {{- if or (not .Values.persistence.enabled) (not (eq .Values.persistence.provider "local")) }}
+  revisionHistoryLimit: {{ .Values.revisionHistoryLimit }}
+  {{- end }}
+  {{- with .Values.priorityClassName }}
+  priorityClassName: {{ . }}
+  {{- end }}
```

Comment:
Isn't this a `pod` specification instead of a `deployment` specification? It should therefore be defined in `spec.template.spec`.

Reply:
you are correct, fixed!